lib/fs.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+// TODO: this file should be removed and `fs.promises` used instead
+// when support for Node.js 8 is dropped
+
+const fs = require('fs');
+const { promisify } = require('util');
+
+const { iter } = require('@metarhia/common');
+
+const list = [
+  'readFile',
+  'writeFile',
+  'unlink',
+  'rename',
+  'stat',
+  'access',
+  'mkdir',
+  'rmdir',
+  'readdir',
+];
+
+if (process.version.slice(1).split('.')[0] >= 10) {
```

Comment:
I think we should even put '11.14.0' (https://github.com/nodejs/node/pull/26581) in here to avoid having the warning in the console every time this is run.